src/types/PdfIngestion.ts

```diff
@@ -0,0 +1,18 @@
+export interface PdfIngestionResponse {
```

Comment:
I would move this to the same new file/folder as we would move the pdf ingestion methods in filesystem.ts